queries/tact/folds.scm

```diff
@@ -0,0 +1,14 @@
+[
+  ; (…, …)
+  (parameter_list)
+  (argument_list)
+  ; {…, …}
+  (instance_argument_list)
+  ; {…; …}
+  (message_body)
+  (struct_body)
+  (contract_body)
+  (trait_body)
+  (function_body)
+  (block_statement)
```

Comment:
Would you mind adding `(import_statement)+` here?

Reply:
Sure, one moment